optuna/integration/chainer.py

```diff
@@ -28,15 +28,12 @@ class ChainerPruningExtension(Extension):
 
     Example:
 
-        Add a pruning extension which observes validation losses to
+        See `the example <https://github.com/optuna/optuna/blob/30dba8814d332ff4
+        0d9c7465ee660a8ecf499547/examples/pruning/chainer_integration.py>`__
```

Comment:
I believe it's more user friendly to reference the latest example and not a particular snapshot. This documentation must be maintained in accordance to the example scrip in any way. This is not a strong opinion though as I see that this is consistent with earlier similar changes, https://github.com/optuna/optuna/pull/1019.

Reply:
I agree.
I think it will be much more maintenance costs if we use specific commit in the URL.
We can avoid this by using master branch. (For example, we can see this usage in [TensorFlow Document](https://github.com/tensorflow/docs/blob/master/site/en/guide/profiler.md#install-the-profiler-and-gpu-prerequisites) .)
However, it also has the disadvantage described [here](https://github.com/optuna/optuna/pull/1019#discussion_r395010625).

I think using master branch in example URL is better because of simple and low maintenance cost.

Could you give me any advises about this?

cc: @himkt

Reply:
I wrote the comment based on [the review which I received previously](https://github.com/optuna/optuna/pull/952#discussion_r384866934).
In #952, we have to refer to the implementation which is removed in the master branch.
However, in this PR, it would be better to refer to the implementation of the master branch.
So I agree with @hvy and @nuka137 to remove commit id and refer to the master branch.

@nuka137 I'm really sorry for confusing you.

Reply:
Sorry for reraising a sort of settled issue. I believe however that the consensus to here refer to the master is reasonable. @nuka137 could you update the changes in this PR?

Reply:
Thanks!
OK, I will fix them (include examples/pruning/tensorflow_estimator_integration.py as well).

Reply:
@hvy @himkt 

Thanks for your advice!
I fixed all example links to refer master branch.